app/views/visitor_mailer/_multiple_cancellation_reasons.html.erb

```diff
@@ -10,3 +10,4 @@
 <ul class="list list-bullet">
   <%= render @cancellation.formatted_reasons %>
 </ul>
+<p><%= t('arrange_html', scope: %i(visitor_mailer cancelled)) %></p>
```

Comment:
might be wrong but is this not missing a `.` in front of `arrange_html`?

Reply:
I don't think you use a . when adding a scope? Not sure, was just copying what was already there.

Reply:
ok, it's probably fine then. What could possibly go wrong? :trollface: